src/services/rfqm_service.ts

```diff
@@ -204,10 +205,13 @@ export class RfqmService {
         const makerAmountInUnit = Web3Wrapper.toUnitAmount(bestQuote.makerAmount, makerTokenDecimals);
         const takerAmountInUnit = Web3Wrapper.toUnitAmount(bestQuote.takerAmount, takerTokenDecimals);
         const price = isSelling ? makerAmountInUnit.div(takerAmountInUnit) : takerAmountInUnit.div(makerAmountInUnit);
+        // The way the BigNumber round down behavior (https://mikemcl.github.io/bignumber.js/#dp) works requires us
+        // to add 1 to PRICE_DECIMAL_PLACES in order to actually come out with the decimal places specified.
+        const roundedPrice = price.decimalPlaces(PRICE_DECIMAL_PLACES + 1, BigNumber.ROUND_DOWN);
 
         // Prepare response
         return {
-            price,
+            price: roundedPrice,
```

Comment:
Should this be a number or a string?

Reply:
The type was `BigNumber` and remains `BigNumber`